tests/api/collections/test_collections_views.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests Views for collections."""
+
+
+from rero_ils.modules.collections.api import Collection
+from rero_ils.modules.collections.views import get_document, get_items, \
+    get_location, get_teachers, start_end_date
+
+
+def test_start_end_date(db, coll_martigny_1_data):
```

Comment:
I think this test can be moved to the `tests/ui/collections` directory

Reply:
Done